packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -688,3 +690,65 @@ export function isParentOf(element, targetElement) {
     false
   );
 }
+
+/**
+ * Copies text to clipboard using the Clipboard API.
+ * https://developer.mozilla.org/en-US/docs/Web/API/Clipboard
```

Comment:
Maybe just a tad bit of information as to why this navigator API would fail?

Also does that work on mobile?

Reply:
Sure yeah, I will try investigate why it can fail and explain this in a comment. cause for now I just copied the logic that we had in the QuanticError component directly. 
Yes this works on mobile👌